### DISCUSSION TITLE
Any plans for windows entry at conda-forge?

### DISCUSSION THREAD

**Discussion by andife** [ImageMagick/ImageMagick#5628] [General]
Hello,
I wonder if there are any plans to add windows at
https://anaconda.org/conda-forge/imagemagick
?

**Comment by snibgo** [comment]
That seems to be a question for people at Anaconda.

**Reply by andife** [reply]
Not necessarily. conda-forge itself is community driven

conda-forge (https://conda-forge.org/)
"A community-led collection of recipes, build infrastructure and distributions for the conda package manager."

**Reply by dlemstra** [reply]
We don't have any plans to add Windows there and we are not maintaining that package so you should probably ask someone else.
